src/main/resources/hudson/plugins/copyartifact/Messages.properties

```diff
@@ -1,3 +1,4 @@
+CopyArtifact.AncestorIsNull=Ancestor/Context Unknown: the project specified cannot be validated
```

Comment:
Well it is not exactly “non-buildable”. But certainly the term “ancestor” will be meaningless to a user (this is an API term). I think it suffices to say that the configuration form is being displayed without enough context to determine which job will host the build step.